### PR TITLE
PIM-5028 : fix log level configuration

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                 | A
| ----------------- | ---
| Specs             |no additional test
| Behats            |no additional test
| Blue CI           |running
| Changelog updated |yes
| Review and 2 GTM  |ok

### CODE REVIEW

**Comment by solivier** [comment]
GTM

**Comment by phaseinducer** [comment]
GTM

**Comment by grena** [comment]
GTM

File: app/config/config_dev.yml [hunk 1]
```diff
@@ -12,7 +12,8 @@ web_profiler:
 monolog:
     handlers:
         main:
-            type:  stream
+            type: rotating_file
```
Comment (jjanvier) [inline app/config/config_dev.yml]: isn't it better to let do this with logrotate?
I'm a big fan of this kind of "we do things in PHP that are already done by something else"

File: app/config/config_dev.yml [hunk 2]
```diff
@@ -12,7 +12,8 @@ web_profiler:
...
+            max_files: 10
```
Comment (jjanvier) [inline app/config/config_dev.yml]: it's not good to delete logs older than 10 days IMO

File: app/config/config_dev.yml (hunk 1 above)
Comment (jmleroux) [inline app/config/config_dev.yml]: it is just the dev log.
To avoid ultra fat log files.

Comment (jjanvier) [inline app/config/config_dev.yml]: ok it's for dev, sorry

**Comment by jjanvier** [comment]
GTM

File: app/config/config_dev.yml (hunk 1 above)
Comment (BitOne) [inline app/config/config_dev.yml]: No rotating file please.
In prod: logrotate
In dev: cleanup your log yourself

**Comment by BitOne** [comment]
GTM now ;)

**Pull request merged**